vars/testsInStage.groovy

```diff
@@ -25,6 +25,11 @@ boolean call() {
                              exit 0
                          fi
                          cd src/tests/ftest
-                         ./list_tests.py """ + parseStageInfo()['test_tag'],
+                         if [ -x list_tests.py ]
```

Comment:
Yeah, oops on my part.  I suggested `-e` but I really did mean `-x`.  Thanks for picking up on that.